core/src/main/scala/pureconfig/package.scala

```diff
@@ -255,9 +256,10 @@ package object pureconfig {
    * @param outputStream The stream in which the configuration should be written
    */
   def saveConfigToStream[Config](conf: Config, outputStream: OutputStream)(implicit writer: Derivation[ConfigWriter[Config]]): Unit = {
-    val printOutputStream = new PrintStream(outputStream)
+    // HOCON requires UTF-8
```

Comment:
Can you support this claim by adding a link to https://github.com/lightbend/config/blob/master/HOCON.md#unchanged-from-json on this comment?

Reply:
Done.